composables/use-issues-by-column.ts

```diff
@@ -0,0 +1,40 @@
+import type { ColumnTitle } from '~/constants/columns'
+import type { IssueStatus } from '~/constants/issues'
+import type { Issue } from '~/types/issues'
+
+function filterIssuesWithoutStatus(issues: Issue[]): Issue[] {
+  const issuesWithoutStatus = issues.filter(
+    (issue) =>
+      !issue.labels.some((label) => label.startsWith(createPartialDataLabel('status'))),
+  )
+
+  return issuesWithoutStatus
+}
+
+function filterIssuesByStatus(issues: Issue[], status: IssueStatus): Issue[] {
+  const issuesWithStatus = issues.filter((issue) =>
+    issue.labels.includes(createDataLabel('status', status)),
+  )
+
+  return issuesWithStatus
+}
+
+function useIssuesByColumn(): Record<ColumnTitle, ComputedRef<Issue[]>> {
+  const route = useRoute('node-rid')
+  const response = useHttpdFetch('/projects/{rid}/issues', {
+    path: {
+      rid: route.params.rid,
+    },
+  })
+
+  const issues = response.data as Ref<Issue[] | undefined>
+
+  return {
+    'not-planned': computed(() => filterIssuesWithoutStatus(issues.value ?? [])),
+    'todo': computed(() => filterIssuesByStatus(issues.value ?? [], 'todo')),
+    'doing': computed(() => filterIssuesByStatus(issues.value ?? [], 'doing')),
+    'done': computed(() => filterIssuesByStatus(issues.value ?? [], 'done')),
+  }
+}
+
+export default useIssuesByColumn
```

Comment:
Please don't use `export default`... as a default :P Makes global-searching and especially renaming symbols across files a pain.

Prefer named exports unless there's good reason not to.

Reply:
👍